compass/ocean/tests/spherical_harmonic_transform/qu_convergence/init.py

```diff
@@ -0,0 +1,68 @@
+from compass.model import run_model
+from compass.step import Step
+
+
+class Init(Step):
+    """
+    A step for running a spherical harmonic transformation
+    for the shperical_harmonic_transfrom test case
+    """
+    def __init__(self, test_case, resolution, algorithm, order):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.spherical_harmonic_transform.qu_convergence.init
+            The test case this step belongs to
+
+        resolution : int
+            The resolution of the (uniform) mesh in km
+
+        algorithm : str
+            Use either the 'parallel' or 'serial' algorithm
+
+        order : int
+            For algorithm = 'parallel', the order of the shperical harmonic transform
+            For algorithm = 'serial', the number of latitudes in the Gaussian grid
+        """
+        super().__init__(test_case=test_case,
+                         name=f'QU{resolution}_init_{resolution}_{order}',
+                         subdir=f'QU{resolution}/init/{algorithm}/{order}',
+                         cores=36, min_cores=1)
+
+        package = 'compass.ocean.tests.spherical_harmonic_transform.qu_convergence'
```

Comment:
For future test groups/cases, this would be a reason to choose a shorter name if possible.

Reply:
it's not too late! I'm happy to change it.

Reply:
I'll leave it up to you.

Reply:
Otherwise, it's a little awkward and maybe isn't enough but:
```suggestion
        package = \
            'compass.ocean.tests.spherical_harmonic_transform.qu_convergence'
```